client/Pages/EventTypeDetails/View.elm

```diff
@@ -715,6 +724,32 @@ renderSubscription name appsInfoUrl item =
         ]
 
 
+consumingQueriesPanel : Stores.ConsumingQuery.Model -> Html Msg
+consumingQueriesPanel queryStore =
+    let
+        sqlQueriesList =
+            Store.items queryStore
+
+        count =
+            List.length sqlQueriesList
+
+        countStr =
+            pluralCount count "consuming nakadi-sql queries"
```

Comment:
It produces "consuming nakadi-sql queries**s**" for multiple queries.
This function doesn't support this case "1=y, N=es", you probably need custom code with IF here.

Reply:
Fixed